cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -520,6 +520,11 @@ std::unordered_map<std::string, std::string> WholeStageResultIterator::getQueryC
 
     configs[velox::core::QueryConfig::kSparkPartitionId] = std::to_string(taskInfo_.partitionId);
 
+    // Enable Spark legacy date formatter if spark.sql.legacy.timeParserPolicy is set to 'LEGACY'.
+    if (veloxCfg_->get<std::string>(kSparkLegacyTimeParserPolicy, "") == "LEGACY") {
+      configs[velox::core::QueryConfig::kSparkLegacyDateFormatter] = "true";
```

Comment:
Do we need to set this config as false if 'kSparkLegacyTimeParserPolicy' is not 'LEGACY'?

Reply:
Thanks for your feedback. It is necessary to do this. Updated.